composer.json

```diff
@@ -9,8 +9,9 @@
         "findologic/xml-export-schema": "^1.4"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.5|^8.5|^9.0",
-        "squizlabs/php_codesniffer": "^3.5"
+        "phpunit/phpunit": "^7.5|^8.6|^9.0",
+        "squizlabs/php_codesniffer": "^3.5",
+        "nikic/php-parser": "^4.15"
```

Comment:
:information_source: This is a transitive dependency of PHPUnit used for code coverage processing. When updating PHPUnit with PHP 8.1 and `--prefer-lowest`, the oldest version of this dependency being installed is not PHP 8.1 compliant. Requiring the latest 4.x version works with all supported PHP versions.